src/api/app/models/bs_request.rb

```diff
@@ -58,9 +58,8 @@ class BsRequest < ApplicationRecord
   scope :by_package_reviews, ->(package_ids) { where(reviews: { package: package_ids }) }
   scope :by_group_reviews, ->(group_ids) { where(reviews: { group: group_ids }) }
 
-  scope :for_user, ->(params) { BsRequest::FindFor::User.new(params).all }
```

Comment:
Those removed scopes weren't used, beside in `BsRequest::FindFor::Query`, so it's safe to remove them.